src/test/java/RowLevelSecurityIT.java

```diff
@@ -18,7 +18,7 @@
     private static final String DATABASE_USER = "SYS";
     private static final String DATABASE_PWD = "exasol";
 
-    @Test
+
```

Comment:
Test marker gone.

Reply:
That's because I deactivated them. They are integration tests that require container

Reply:
Use switch via annotiation in this case please.